Ryujinx.Common/Configuration/AppDataManager.cs

```diff
@@ -81,6 +89,14 @@ public static void Initialize(string baseDirPath)
 
             BaseDirPath = Path.GetFullPath(BaseDirPath); // convert relative paths
 
+            // Copy .config dir to application support dir
+            if (System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.OSX) && Mode == LaunchMode.UserProfile)
+            {
+                var oldConfigPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), DefaultBaseDir);
+                if (Path.Exists(oldConfigPath) && !Path.Exists(BaseDirPath))
+                    CopyDirectory(oldConfigPath, BaseDirPath);
```

Comment:
```suggestion
                if (Path.Exists(oldConfigPath) && !Path.Exists(BaseDirPath))
                {
                    CopyDirectory(oldConfigPath, BaseDirPath);
                }
```
You didn't delete or rename `oldConfigPath` which could be nice too.